### PR TITLE
Day2 team work

### PR DESCRIPTION
**Pull request by az-16**
Artur and Olena please say something to my code. How can I improve?

### CODE REVIEW

File: src/test/java/com/cybertek/aizat_work_here/Hello.java [hunk 1]
```diff
@@ -1,5 +1,12 @@
 package com.cybertek.aizat_work_here;
 
+
+
 public class Hello {
+    public static void main(String[] args) {
+
```
Comment (ArturZalutskyi) [inline src/test/java/com/cybertek/aizat_work_here/Hello.java:7]: why here is emty space???????????????!!!!!!!!!!!!!!!!!!

**Review by ArturZalutskyi — COMMENTED** [review]
Now you supposed to see it
zoom
zoom???

File: src/test/java/com/cybertek/aizat_work_here/Hello.java (hunk 1 above)
Comment (az-16) [reply src/test/java/com/cybertek/aizat_work_here/Hello.java:7]: Oh I will remove empty space))

**Review by az-16 — COMMENTED** [review]
Changed empty space

**Comment by ArturZalutskyi** [comment]
good jo

**Pull request merged**